src/components/Panel/PanelContainer.js

```diff
@@ -112,6 +118,10 @@ export default class PanelContainer extends Component {
     });
   }
 
+  panelInstanceId = `panel-${getInstanceId()}`;
```

Comment:
This is minor, but can we omit the `panel` prefix here from the static variable names (`panelInstanceId`, etc.) as it's already implied that we're in the scope of the `Panel` as per the `footer` and `header` below?

```suggestion
  instanceId = `${namespace}--${getInstanceId()}`;
```

Reply:
I can definitely update some of the variable names, but `panel-${getInstanceId()}` adds is a bit different from `${namespace}--${getInstanceId()}`. Namespace is already getting added later. So with the first you end up with a nice id like `security--panelv2__title--panel-1`, and with the second you get something longer like `security--panelv2__title--security--panelv2-1`

Reply:
Oh! I wonder in that case is it enough to remove the `panel` prefix entirely?

Reply:
I'm okay with either way, but honestly I kinda like it reading like "panel 2" or whatever, in the modifier.